extensions/kind/src/extension.spec.ts

```diff
@@ -37,68 +39,78 @@ vi.mock('./image-handler', async () => {
   };
 });
 
-vi.mock('@podman-desktop/api', async () => {
-  return {
-    window: {
-      withProgress: vi.fn(),
-      createStatusBarItem: vi.fn(),
-    },
-    cli: {
-      createCliTool: vi.fn(),
-    },
-    ProgressLocation: {
-      TASK_WIDGET: 'TASK_WIDGET',
-    },
-    provider: {
-      onDidRegisterContainerConnection: vi.fn(),
-      onDidUnregisterContainerConnection: vi.fn(),
-      onDidUpdateProvider: vi.fn(),
-      onDidUpdateContainerConnection: vi.fn(),
-      createProvider: vi.fn(),
-    },
-    containerEngine: {
-      listContainers: vi.fn(),
-      onEvent: vi.fn(),
-    },
-    commands: {
-      registerCommand: vi.fn(),
-    },
-    context: {
-      setValue: vi.fn(),
-    },
-    env: {
-      isWindows: false,
-      isMac: false,
-      isLinux: true,
-      createTelemetryLogger: vi.fn().mockReturnValue({
-        logUsage: vi.fn(),
-      } as unknown as extensionApi.TelemetryLogger),
-    },
-    process: {
-      exec: vi.fn(),
-    },
-  };
-});
+vi.mock('./kind-installer', () => ({
+  KindInstaller: vi.fn(),
+}));
+
+vi.mock('@podman-desktop/api', () => ({
+  window: {
+    withProgress: vi.fn(),
+    createStatusBarItem: vi.fn(),
+  },
+  cli: {
+    createCliTool: vi.fn(),
+  },
+  ProgressLocation: {
+    TASK_WIDGET: 'TASK_WIDGET',
+  },
+  provider: {
+    onDidRegisterContainerConnection: vi.fn(),
+    onDidUnregisterContainerConnection: vi.fn(),
+    onDidUpdateProvider: vi.fn(),
+    onDidUpdateContainerConnection: vi.fn(),
+    createProvider: vi.fn(),
+  },
+  containerEngine: {
+    listContainers: vi.fn(),
+    onEvent: vi.fn(),
+  },
+  commands: {
+    registerCommand: vi.fn(),
+  },
+  context: {
+    setValue: vi.fn(),
+  },
+  env: {
+    isWindows: false,
+    isMac: false,
+    isLinux: true,
+    createTelemetryLogger: vi.fn(),
+  },
+  process: {
+    exec: vi.fn(),
+  },
+}));
 
-const kindInstallerMock = {
+const KIND_INSTALLER_MOCK: KindInstaller = {
   getLatestVersionAsset: vi.fn(),
   getKindCliStoragePath: vi.fn(),
   download: vi.fn(),
   promptUserForVersion: vi.fn(),
 } as unknown as KindInstaller;
```

Comment:
AFAIK with vi.mock('./KindInstaller') we don't need that

Reply:
Applied!